src/AspNet.Security.OpenIdConnect.Server/project.json

```diff
@@ -21,8 +21,8 @@
         },
 
         "Microsoft.Framework.WebEncoders.Core": "1.0.0-*",
-
-        "Newtonsoft.Json": "7.0.1"
+        "Newtonsoft.Json": "7.0.1",
+        "System.Dynamic.Runtime": "4.0.0"
```

Comment:
Hum, weird. If you really need it, please move it under `dnxcore50`, since we don't want it for `dnx451`.